src/molecules/lists/unordered-list.tsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import { Icons } from "../../atoms/constants/icons";
+
+// -----------------------------------------------------------------------------------------
+// #region Interfaces
+// -----------------------------------------------------------------------------------------
+
+export interface UnorderedListProps {
+    cssClassName?: string;
+    id?: string;
+    listItems: Array<any>;
+    listIcon?: keyof typeof Icons;
```

Comment:
Question - does this need to be `keyof typeof` or can it just be typed as `Icons`?

Reply:
Yeah, being an enum it doesn't need keyof typeof